node/service/src/rpc/tests/mod.rs

```diff
@@ -0,0 +1,55 @@
+mod rpc_mock;
+
+use rpc_mock::*;
+
+use crate::rpc::frequency_rpc::{FrequencyRpcApiServer, FrequencyRpcHandler};
+use codec::{Decode, Encode};
+use common_primitives::rpc::RpcEvent;
+use frame_system::{EventRecord, Phase};
+use sp_core::H256;
+use sp_runtime::scale_info::TypeInfo;
+use std::sync::Arc;
+use substrate_test_runtime_client::runtime::Block;
+use system_runtime_api::AdditionalRuntimeApi;
+
+#[derive(Debug, TypeInfo, Clone, Encode, Decode, Eq, PartialEq)]
+enum FakeRuntime {
+	#[codec(index = 60)]
+	FakePallet(FakePalletEvents),
+}
+
+#[derive(Debug, TypeInfo, Clone, Encode, Decode, Eq, PartialEq)]
+enum FakePalletEvents {
+	#[codec(index = 7)]
+	FakeEvent { msa_id: u64 },
```

Comment:
nitpick - it's generally better to test N>1 things, and 3 is more N-like than 2.   I would want to test that N>1 events and N>1 pallets output worked.  Though it does work fine now, so maybe for later.

Reply:
Good point, but agreed that it is a maybe later action. I want to get some feedback on it first.